plugins/modules/source_control/gitlab/gitlab_group.py

```diff
@@ -197,23 +197,27 @@ def get_group_id(self, group):
     def create_or_update_group(self, name, parent, options):
         changed = False
 
+        # Same payload for update or create, only parent_id is needed on creation
+        payload = {
+            'name': name,
+            'path': options['path'],
```

Comment:
So far `path` was only passed on creation, I would handle it the same as `parent_id`.